src/extension.ts

```diff
@@ -45,11 +44,12 @@ export function activate(context: ExtensionContext) {
       revealOutputChannelOn: RevealOutputChannelOn.Never,
       initializationOptions: { settings: getSettings() },
       synchronize: {
-        configurationSection: ['xml']
+        configurationSection: ['xml', '[xml]']
```

Comment:
is this required? The server doesn't need to get synchronize on [xml] I think

Reply:
It allows for [specific language configurations](https://github.com/redhat-developer/vscode-xml/pull/62/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R155) to be considered for didChangeConfiguration.